bhclub.html

```diff
@@ -32,6 +32,7 @@
         </div>
         <div>
             <a class="social-link" href="https://discord.gg/gNJXjvzkuB" target="_blank">Discord</a>
+            <a class="social-link" href="#" target="_blank">CampusGroups</a>
```

Comment:
I think Michael said in the BrickHack server that we don't need to include a CampusGroups link if you don't want to.

Reply:
At this point I'm leaving decision of what links are actually included for the final pr (#1206), since we have about five different lists and I've already got a ton of merge conflicts with this and the about pr